src/views/Home/index.js

```diff
@@ -295,7 +295,9 @@ const Home = () => {
             <Grid className={classes.gridContainer} container>
                 <Grid item md={12} className={classes.gridChild}>
                     <Box >
-                        <LobbyRoom tracks={localTracks}/>
+                        {
+                            localTracks?.length && <LobbyRoom tracks={localTracks}/>
```

Comment:
@brajendrak00068 When was this reverted. This is what i was talking about earlier we have done the similar changes